packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -284,12 +295,37 @@ export class TaggedLoggerAdapter implements ITelemetryBaseLogger {
 	}
 }
 
+const childLoggerType = "__fluid_child_logger" as const;
+
+export function createChildLogger(props?: {
+	logger?: ITelemetryBaseLogger;
+	namespace?: string;
+	properties?: ITelemetryLoggerPropertyBags;
+}): ITelemetryLoggerExt {
+	if (props === undefined) {
+		return new TelemetryNullLogger();
+	}
+	return ChildLogger.create(props?.logger, props?.namespace, props?.properties);
```

Comment:
```suggestion
	return ChildLogger.create(props.logger, props.namespace, props.properties);
```

Or, just remove the `if` check above and let there be one codepath since apparently `ChildLogger` can do it.  And remove `TelemetryNullLogger` altogether